pkgs/applications/audio/rofi-mpd/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, python37Packages, fetchFromGitHub, pkgs }:
+
+with python37Packages; python37Packages.buildPythonPackage rec {
+  pname = "rofi-mpd";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "JakeStanger";
+    repo = "Rofi_MPD";
+    rev = "168f3e7205761c960abd9b44c83f109cefcb9c2c";
+    sha256 = "0pdra1idgas3yl9z9v7b002igwg2c1mv0yw2ffb8rsbx88x4gbai";
+  };
+
+  propagatedBuildInputs = [ mutagen mpd2 ];
+
+  meta = with stdenv.lib; {
+    description = "A rofi menu for interacting with MPD written in Python";
+    homepage = "https://github.com/JakeStanger/Rofi_MPD";
+    license = licenses.mit;
+    maintainers = with maintainers; [ jakestanger ];
+    platforms = platforms.all;
+  };
+
+  doCheck = false;
```

Comment:
any particular reason why not to include tests?

Reply:
When `unittest` runs it passes arguments to `argparse` which throws up that the arguments are invalid and returns code 2.

Reply:
if you do want to include tests you could also override checkPhase
```
 checkPhase = ''
    ${python.interpreter} -m unittest test_module
 '';
```

Reply:
Noted. I don't have any unit tests for the package at the moment but I'll do this if I need to in the future.